precommits/pre-commit

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+ROOTPATH=$(git rev-parse --show-toplevel)
+
+if [ $# -eq 0 ]; then
+	files=$(git diff --cached --name-only --diff-filter=AM)
+else
+	files=$@
+fi
+
+pattern="src.*\.ts$"
+status=0
+for file in $files
+do
+	if [[ ${file} =~ $pattern ]]; then
```

Comment:
`index.ts` it's outside `src` folder. We should move it to a folder under src so everything is run under the hooks or change the pattern of the hooks to consider files outside that folder.

Reply:
Example:
```bash
diff --git a/index.ts b/index.ts
index f885c83..af94c74 100644
--- a/index.ts
+++ b/index.ts
@@ -3,7 +3,7 @@ import { start } from "./server/start";
 start()
   .then(() => {
     // tslint:disable-next-line:no-console
-    console.log("Connected!");
+    console.log('Connected!');
   }).catch((err) => {
     // tslint:disable-next-line:no-console
     console.error(`Error starting server: ${err.message}`);
```
--------------------------------------------------------
Prehooks not jumping!

```bash
gcmsg "test"                                                                                                                     [00:17:13]
[precommits 1ec2a8c] test
 1 file changed, 1 insertion(+), 1 deletion(-)
```

Reply:
Correct. The index.ts isn’t included yet. I’ll do the change tonight